readme.MD

```diff
@@ -1,57 +1,106 @@
-
-To quickly see our project in action, visit the [live demo](https://repl.it/@gurbuzertunga/bubblesort) on repl.it.
-
-## Getting Started
-
-### Prerequisites
-
-To get this project up and running, you must already have ruby installed on your computer.
-
-### Installation
-
-**To get this project set up on your local machine, follow these simple steps:**
-
-1. Open Terminal.
-2. Navigate to your desired location to download the contents of this repository.
-3. Copy and paste the following code into the Terminal:
-    git clone https://github.com/gurbuzertunga/bubble_sort.git
-4. Hit enter.
-5. Once the repository has been cloned, navigate inside the repository and type:
-    ruby bubble_sort.rb
-    This will run the code, printing the output of our tests to the terminal.
-6. To change those tests, simply open bubble_sort.rb in a code editor and modify the values stored in the test_array and test_run variables.
-6. Enjoy sorted arrays of data for the rest of your life.
-
-## Authors
-
-:bust_in_silhouette: **Mahmoud Muhammad**
-- Github: [@mahmoud717](https://github.com/mahmoud717)
-- Twitter: [@mahmoud](https://twitter.com/mahmoud26369406)
-
-:bust_in_silhouette:**Gürbüz Ertunga**
-- Github: [@gurbuzertunga](https://github.com/gurbuzertunga)
-- LinkedIn: [@Gurbuz Ertunga](https://www.linkedin.com/in/gurbuz-ertunga-a607a2a5/)
-## :handshake: Contributing
-
-## Show your support
-If you've read this far, you must like the project! Give us a :star:️!
-## Acknowledgments
-- This Project was part of an assignment available on The Odin Project.
-- Our thanks to Microverse and all our peers and colleagues there.
-## :memo: License
+# The Tic-Tac-Toe
+
+> In this project, we are building a Tic Tac Toe game. The main goal is to put into practice the main concepts of Object Oriented Programming. Particularly, classes, modules and objects and access their attributes.
+
+For this project, we wrote a Ruby executable script main.rb that allows the user to play the game of Tic, Tac, Toe in the Terminal.
+
+## How To Play
+
+'Tic, Tac, Toe' is a traditional game for two players played on a board with 9 squares.
+
+Each player is represented by a symbol, either 'X' or 'O'. The players then take turns to place their respective symbols in an empty square on the board, with O's going first - in our version of the game, these squares are represented by the numbers 1 to 9.
+
+The aim of the game is to have three of your symbols form a line, be it vertically, horizontally or diagonally, as illustrated in the image below.
+
+![A Winning Combination of Tic, Tac, Toe](.github/winning-screenshot.webp)
+
+In the event that all 9 squares are taken without either player getting three of their symbols in a line, a draw is declared.
+
+### Accessing The Game
+
+- To quickly play the game, follow the link in the [Live Demo section](https://repl.it/@gurbuzertunga/tic-tac-toe#bin/main.rb) and hit the Run button in the header. The game will begin in the window on the right.
```

Comment:
- [ ] I'm sorry, but this instruction doesn't seem to work properly. Please provide a working link or add configuration on .replit instructions.
**Hint:** For more complex projects with many directories, I would suggest you (and this solution is optional) to use Gitpod, just append `gitpod.io/` to the URL of your repo:
https://gitpod.io/github.com/gurbuzertunga/tic-tac-toe/tree/readme_game_instructions

![image](https://user-images.githubusercontent.com/53256139/91447075-e0c6de80-e83d-11ea-8b07-4f5209e837f6.png)